src/test/java/unittest/testcases/math/MathsTests.java

```diff
@@ -287,4 +287,15 @@ void tryDenormalization_usingFloatingPointMath_butProvidedNumberIsLargerThanOne(
         String expectedErrorMessage = "The normalized number must be less than 1.0f.";
         assertEquals(expectedErrorMessage, exception.getMessage(), "The function call should throw an error, and the error message should match the expected error message.");
     }
+
+    @Test
+    void checkInverseLerpValues() {
+        float testOne = Maths.inverseLerp(10f, 20f, 10.0f);
+        float testTwo = Maths.inverseLerp(10f, 20f, 19.6f);
+        float testThree = Maths.inverseLerp(10f, 20f, 20.0f);
+
+        assertEquals(testOne, 0.0f, "Expected value was 0.0, instead got " + testOne);
```

Comment:
Each of these unit test assertions have their expected and actual values swapped.
The order should go as follows: (expected, actual, message)